src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -4,9 +4,9 @@
 
 def arbeitsl_geld_2_eink_m(  # noqa: PLR0913
     arbeitsl_geld_2_bruttoeink_m: float,
-    eink_st_y_tu: float,
-    soli_st_y_tu: float,
-    anz_erwachsene_tu: int,
+    eink_st_y_sn: float,
+    soli_st_y_sn: float,
+    anz_personen_sn: int,
     sozialv_beitr_m: float,
     arbeitsl_geld_2_eink_anr_frei_m: float,
     kind: bool,
```

Comment:
```suggestion
```

The conditional expression can just go, I think. Kids have their own `sn` and we calculate income taxes for them -- all good.